FamilySearchIosSampleApp/MemoriesVC.swift

```diff
@@ -52,7 +52,7 @@ class MemoriesVC : UICollectionViewController
         configuration.HTTPAdditionalHeaders = headers;
         let session = NSURLSession(configuration: configuration)
         
-        let memoriesTask = session.dataTaskWithURL(NSURL(string:user.artifactsHref!)! ) { (memoriesData, response, memoriesError) in
+        let memoriesTask = session.dataTaskWithURL(NSURL(string:user!.artifactsHref!)! ) { [weak self] (memoriesData, response, memoriesError) in
```

Comment:
It would be good to extract the NSURL construction out to a guard so we don't force unwrap things.